gke-disk-image-builder/cli/main.go

```diff
@@ -55,6 +55,7 @@ func main() {
 	serviceAccount := flag.String("service-account", "default", "Service Account email assigned to the GCE instance used for creating the disk image.")
 	diskType := flag.String("disk-type", "pd-ssd", "disk type to generate the disk image")
 	diskSizeGb := flag.Int64("disk-size-gb", 60, "disk size to unpack container images")
+	instanceDiskSizeGb := flag.Int64("instance-disk-size-gb", 100, "disk size of the GCE instance used for disk image creation")
```

Comment:
please add logic for if instanceDiskSizeGb is not specified, use the value from `diskSizeGb`